src/components/common/AddEditOrgPerson.vue

```diff
@@ -115,13 +115,10 @@
 
             <!-- Roles -->
             <article class="mt-2">
-              <div class="font-weight-bold">{{ getPeopleAndRolesResource.rolesTitle || 'Roles' }}</div>
-              <div v-if="getPeopleAndRolesResource.rolesSubtitle">
-                {{ getPeopleAndRolesResource.rolesSubtitle }}
-              </div>
+              <label>Roles</label>
               <div class="form__row three-column mt-4">
                 <v-card flat rounded="sm" class="item gray-card px-4">
-                  <v-row no-gutters class="align-center mt-4">
+                  <v-row no-gutters class="align-center mt-5">
```

Comment:
Consistent whitespace, ie.

![image](https://user-images.githubusercontent.com/10002889/213328621-31884283-a90e-4fe8-ad51-756043d54332.png)

Reply:
@JazzarKarim copy the code for this ^^